components/datetime/src/calendar.rs

```diff
@@ -251,6 +264,23 @@ impl CldrCalendar for Roc {
     type MonthSymbolsV1Marker = RocMonthSymbolsV1Marker;
 }
 
+impl UnstableCldrCalendar for Buddhist {}
```

Comment:
hmm, this will show up on docs, which we build with all features, and look like `Buddhist` is an unstable calendar. Maybe call this `InternalCldrCalendar`?